containers/Ideas/Idea.js

```diff
@@ -15,67 +16,97 @@ export const Idea = (props) => {
     params.labels.length > 0 && labelsList.push(...params.labels);
     return labelsList;
   };
+  const toggleMenu = () => setMenuVisible(!menuVisible);
+  const [menuVisible, setMenuVisible] = useState(false);
+  const menuItems  = [
+    {
+      title: 'Edit',
+      icon: 'pencil',
+      action: () => console.log('Edit'),
+      isFirst: true
+    },
+    {
+      title: 'Delete',
+      icon: 'trash',
+      action: () => console.log('Delete')
+    },
+    {
+      title: 'Report',
+      icon: 'flag',
+      action: () => console.log('Report'),
+      isLast: true
+    },
+    {
+      title: 'Cancel',
+      action: () => toggleMenu(),
+      isCancel: true
+    },
+  ];
 
   return (
-    <ScrollView
-      style={styles.container}
-      contentContainerStyle={styles.contentContainer}
-    >
-      <View style={styles.actionsContainer}>
-        <Button
-          buttonStyle={styles.backButton}
-          titleStyle={styles.backButtonText}
-          title='Back'
-          type='clear'
-          icon={<Icon name='arrow-left' size={22} />}
-          onPress={() => props.navigation.goBack()}
-        />
-        <Button
-          icon={<Icon name='options-vertical' size={22} />}
-          type='clear'
-        />
-      </View>
-      <View style={styles.titleContainer}>
-        <Text style={styles.title}>{params.name}</Text>
-      </View>
-      <View style={styles.descriptionContainer}>
-        {params.image && (
-          <Image source={{ uri: params.image }} style={styles.ideaImage} />
-        )}
-        <Text style={styles.text}>{params.description}</Text>
-      </View>
-      {getLabels().length > 0 && (
-        <View style={styles.labelsContainer}>
-          {getLabels().map((label, idx) => (
-            <Label key={idx + label} title={label} />
-          ))}
-        </View>
-      )}
-      <View style={styles.infoContainer}>
-        <Text style={styles.creator}>
-          {params.creator} {createdDate}
-        </Text>
-        <Text style={styles.text}>
-          Reference No.: {params.reference_number || 'n/a'}
-        </Text>
-      </View>
-      <View style={styles.bottomActionsContainer}>
-        <View style={styles.ratingButtons}>
-          <ButtonCounter
-            icon={<Icon name='arrow-up' size={18} />}
-            counter={params.positive_rating_count}
+    <>
```

Comment:
except of last line  `<Menu...` it is just indentation

Reply:
Doe you mean how different file is?
Also do we need the console.log?